### ISSUE TITLE
✅ ♿ 🐞 Missing aria-hidden attributes on decorative elements such as Shape Dividers, Background Videos, and Background Images

### ISSUE THREAD

**Issue by rnewhook586** [elementor/elementor#23880]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

### Description

Multiple items are being read by the screen-readers 

1. Shape dividers
2. background videos
3. background images* (generally these are purely decretive and we should have the option to hide with a toggle or something imo)

**For now can be fixed with the following code:**

```html

<!-- Accessability fixes -->
<script>
// Make elements hideden from screen reader
    const elementorShapes = document.querySelectorAll('.elementor-shape , .elementor-background-video-container');
    
    console.log("start setting aria-hidden");
    elementorShapes.forEach( element => {
      element.setAttribute("aria-hidden", "true");
      console.log("updated break");
    });
    console.log("Accessablity hotfixes complete");
</script>
<!-- END CONTENT -->

```

### Steps to reproduce

Navigate through page with Mac OS Voice Over Utility

### Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [x] I can reproduce this bug consistently following the steps above.

### System Info
[system-info-nysdolreports.com-28-09-2023.txt](https://github.com/elementor/elementor/files/12752266/system-info-nysdolreports.com-28-09-2023.txt)

All browsers, Mac OS Voice Over Utility

**Comment by nicholaszein** [comment]
Hello @rnewhook586, 

Thank you for submitting your report! 🙏 

We want to take a look at this issue, but again there is missing information.

Can you please provide the **<ins>FULL, and essential</ins> [System Info](https://elementor.com/help/hare-my-system-info/)** as required in the Bug Report form? This information is not optional.

#### ⚠️ As a gentle reminder, issues without it may need to be closed without troubleshooting. Providing this information upfront aligns with our guidelines, saves time, and ensures swift resolution. Thank you for understanding.

Best regards

**Comment by rnewhook586** [comment]
Added System info to post.

**Comment by rnewhook586** [comment]
@nicholaszein just doing a friendly follow-up on this.

**Comment by rami-elementor** [comment]
Elementor 3.29.0 added `aria-hidden` attribute to **Shape Dividers**.

Ref: https://github.com/elementor/elementor/pull/30621

**Comment by rami-elementor** [comment]
Elementor 3.31.0 will add `aria-hidden` attribute to **Background Videos**.

Ref: https://github.com/elementor/elementor/issues/23880

**Comment by nicholaszein** [comment]
Hello there @rnewhook586!

Elementor v3.31.0 Beta was just released with a fix for this issue! 🙌 

We would appreciate if you could take a look and help us test it to ensure everything is working correctly.

You can check the beta release in the link below:
https://elemn.to/beta-331

To become a beta tester, follow this guide:
https://elemn.to/beta-tester

Kind regards

**Comment by nicholaszein** [comment]
Hello there @rnewhook586!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.29.0 and v3.31.0! 🥳 

Both Shape Dividers and Background Videos now have appropriate `aria` attributes. 

One observation, background images are added via CSS and can't be detected by screen readers, being natively purely decorative already.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂 

----

If you think this is still an issue caused by Elementor after updating to the latest version, and can reproduce it consistently after following our [Contribution Guidelines](https://elemn.to/gh-contributing-bug-reports), feel free to open a new 🐞 [Bug Report](https://elemn.to/gh-new-bug-report), and we will be happy to review your case.
